defaults/main.yml

```diff
@@ -1,9 +1,10 @@
 ---
 gunicorn_start_on: "local-filesystems and net-device-up IFACE!=lo"
-gunicorn_version: "19.2.1"
+gunicorn_version: "19.4.4"
```

Comment:
I'm seeing 19.4.5 as the most recent version. Were you picking 19.4.4 for a specific reason?